source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
```

Comment:
@iamakulov I think this feature should be behind the flag, which is true be default. Seems that some developers in SPA would like to control this manually?